packages/react/src/app-layout/AppLayoutHeader.tsx

```diff
@@ -31,51 +19,69 @@ export type AppLayoutHeaderProps = {
 		/** Dropdown menu for account switching. */
 		dropdown?: ReactNode;
 	};
+	background?: 'body' | 'bodyAlt';
+	/** Used to indicate if an application is in a prerelease state. */
+	badgeLabel?: string;
+	borderColor?: ResponsiveProp<BorderColor>;
+	/** The heading should be set to the website or service title. */
+	heading: string;
+	/**  The href to link to, for example "/". */
+	href: string;
+	/** Defines an identifier (ID) which must be unique. */
+	id?: string;
+	/** The logo to display. */
+	logo: JSX.Element;
+	palette?: ResponsiveProp<BoxPalette>;
+	/** Used to provide additional information to describe your website or service. */
+	subLine?: string;
 };
 
 export function AppLayoutHeader({
-	id,
+	accountDetails,
+	background = 'bodyAlt',
+	badgeLabel,
+	borderColor = 'accent',
 	heading,
 	href,
+	id,
 	logo,
+	palette = 'dark',
 	subLine,
-	badgeLabel,
-	accountDetails,
 }: AppLayoutHeaderProps) {
 	return (
 		<Flex
 			as="header"
-			id={id}
-			palette="dark"
-			flexDirection="column"
 			borderBottom
 			borderBottomWidth="xxl"
-			borderColor="accent"
+			borderColor={borderColor}
 			css={{
 				[tokens.mediaQuery.min[APP_LAYOUT_DESKTOP_BREAKPOINT]]: {
 					gridColumnStart: 1,
 					gridColumnEnd: 3,
 				},
 			}}
+			flexDirection="column"
```

Comment:
Could use `Stack` here. Having said that, would this be better as a `grid`? Column-style flex wrapping a row-style flex suggests 2d layout. Might not be worth it for the amount of changes required?